example/mapboxExampleCamera.js

```diff
@@ -0,0 +1,284 @@
+import * as THREE from 'three';
+
+const utils = {
+	clamp: function ( n, min, max ) {
+
+		return Math.min( max, Math.max( min, n ) );
+
+	},
+	makePerspectiveMatrix: function ( fovy, aspect, near, far ) {
+
+		var out = new THREE.Matrix4();
+		var f = 1.0 / Math.tan( fovy / 2 ),
+			nf = 1 / ( near - far );
+		var newMatrix = [
+			f / aspect,
+			0,
+			0,
+			0,
+			0,
+			f,
+			0,
+			0,
+			0,
+			0,
+			( far + near ) * nf,
+			- 1,
+			0,
+			0,
+			2 * far * near * nf,
+			0,
+		];
+
+		out.elements = newMatrix;
+		return out;
+
+	},
+};
+
+// 512 * 2000
+// 512: TILE_SIZE
+const WORLD_SIZE = 10240000;
+const FOV_ORTHO = ( 0.1 / 180 ) * Math.PI; // Mapbox doesn't accept 0 as FOV
+const FOV = Math.atan( 3 / 4 ); // from Mapbox https://github.com/mapbox/mapbox-gl-js/blob/main/src/geo/transform.js#L93
+const EARTH_RADIUS = 6371008.8; // from Mapbox  https://github.com/mapbox/mapbox-gl-js/blob/0063cbd10a97218fb6a0f64c99bf18609b918f4c/src/geo/lng_lat.js#L11
+const EARTH_CIRCUMFERENCE_EQUATOR = 40075017; // from Mapbox https://github.com/mapbox/mapbox-gl-js/blob/0063cbd10a97218fb6a0f64c99bf18609b918f4c/src/geo/lng_lat.js#L117
+const MapConstants = {
+	WORLD_SIZE: WORLD_SIZE,
+	PROJECTION_WORLD_SIZE: WORLD_SIZE / ( EARTH_RADIUS * Math.PI * 2 ),
+	MERCATOR_A: EARTH_RADIUS,
+	DEG2RAD: Math.PI / 180,
+	RAD2DEG: 180 / Math.PI,
+	EARTH_RADIUS: EARTH_RADIUS,
+	EARTH_CIRCUMFERENCE: 2 * Math.PI * EARTH_RADIUS,
+	EARTH_CIRCUMFERENCE_EQUATOR: EARTH_CIRCUMFERENCE_EQUATOR,
+	FOV_ORTHO: FOV_ORTHO,
+	FOV: FOV,
+	FOV_DEGREES: ( FOV * 180 ) / Math.PI,
+	TILE_SIZE: 512,
+};
```

Comment:
It looks like a number of these constants aren't used in the demo - can we remove the unnecessary ones?

Reply:
ok